demo/prototype-deps.spec

```diff
@@ -0,0 +1,26 @@
+Name:       prototype-deps
+Version:    0.1
+Release:    1%{?dist}
+Summary:    This rpm isn't for real usage and meant to contain only dependencies
+URL:        https://github.com/leapp-to/prototype
+License:    SeeProject
+
+BuildRequires: ansible
+BuildRequires: gcc
+BuildRequires: libguestfs-tools-c
+BuildRequires: libvirt-client
+BuildRequires: libvirt-devel
+BuildRequires: nmap
+BuildRequires: pipsi
+BuildRequires: python2-devel
+BuildRequires: redhat-rpm-config
+BuildRequires: rubygem-hitimes
+BuildRequires: rubygem-nio4r
+BuildRequires: vagrant-libvirt
```

Comment:
This is not really a thing for the bootstrap - I added openssl-devel and python2-pip anyway
but bootstrap needs other things. Anyway I will try to create a copr repo and make a proper prototype rpm in a follow up and then we can see what to do about it - That should make things easier in general.

Anyway for this PR it's not planned to replace the boostrap for the CI just yet - Its meant as a convenience for setting up new dev / demo environments but won't do all the tasks